homeassistant/components/alexa/const.py

```diff
@@ -28,7 +28,9 @@
 
 SYN_RESOLUTION_MATCH = "ER_SUCCESS_MATCH"
 
-DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.0Z"
+# Alexa requires timestamps to be formatted according to ISO 8601, YYYY-MM-DDThh:mm:ssZ
+# https://developer.amazon.com/es-ES/docs/alexa/device-apis/alexa-scenecontroller.html#activate-response-event
```

Comment:
Here is a better link.

```suggestion
# https://developer.amazon.com/en-US/docs/alexa/alexa-voice-service/reportable-state-properties.html#json-representation
```

Reply:
I don't agree your link is "better", it's about state reports whereas the link already there is about activating scenes.
We could maybe include both links?

Reply:
Hmm yeah, I thought I had found a more generic link. Guess they both are about specific messages. It's fine as is